hack/get_ci_vm.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+
+set -e
+
+RED="\e[1;36;41m"
+YEL="\e[1;33;44m"
+NOR="\e[0m"
+USAGE_WARNING="
+${YEL}WARNING: This will not work without local sudo access to run podman,${NOR}
```

Comment:
podman or buildah here?

Reply:
Good catch, nope, `podman` for sure.  It uses a container image I maintain that includes google's cloud tooling and some magic to not mess up file permissions on the host while running.